cmd/convert.go

```diff
@@ -135,15 +145,18 @@ Examples:
 Available Commands:{{range .Commands}}{{if .IsAvailableCommand}}
   {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{ if .HasAvailableLocalFlags}}
 
-Resource Flags:
-      --build-branch             Specify repository branch to use for buildconfig (default is current branch name)
-      --build-repo               Specify source repository for buildconfig (default is current branch's remote url
-  -c, --chart                    Create a Helm chart for converted objects
+Kubernetes Flags:
       --daemon-set               Generate a Kubernetes daemonset object
   -d, --deployment               Generate a Kubernetes deployment object
+  -c, --chart                    Create a Helm chart for converted objects
+      --replication-controller   Generate a Kubernetes replication controller object
+
+OpenShift Flags:
+      --build-branch             Specify repository branch to use for buildconfig (default is current branch name)
+      --build-config             Generate OpenShift BuildConfig
```

Comment:
we should probably remove this flag, as its in `--build` is taking care of that now.